modules/javafx.graphics/src/main/java/com/sun/javafx/tk/quantum/QuantumToolkit.java

```diff
@@ -136,70 +134,58 @@
 
 public final class QuantumToolkit extends Toolkit {
 
-    @SuppressWarnings("removal")
-    public static final boolean verbose =
-            AccessController.doPrivileged((PrivilegedAction<Boolean>) () -> Boolean.getBoolean("quantum.verbose"));
-
-    @SuppressWarnings("removal")
-    public static final boolean pulseDebug =
-            AccessController.doPrivileged((PrivilegedAction<Boolean>) () -> Boolean.getBoolean("quantum.pulse"));
-
-    @SuppressWarnings("removal")
-    private static final boolean multithreaded =
-            AccessController.doPrivileged((PrivilegedAction<Boolean>) () -> {
-                // If it is not specified, or it is true, then it should
-                // be true. Otherwise it should be false.
-                String value = System.getProperty("quantum.multithreaded");
-                if (value == null) return true;
-                final boolean result = Boolean.parseBoolean(value);
-                if (verbose) {
-                    System.out.println(result ? "Multi-Threading Enabled" : "Multi-Threading Disabled");
-                }
-                return result;
-            });
+    public static final boolean verbose = Boolean.getBoolean("quantum.verbose");
 
-    @SuppressWarnings("removal")
-    private static boolean debug =
-            AccessController.doPrivileged((PrivilegedAction<Boolean>) () -> Boolean.getBoolean("quantum.debug"));
+    public static final boolean pulseDebug = Boolean.getBoolean("quantum.pulse");
 
-    @SuppressWarnings("removal")
-    private static Integer pulseHZ =
-            AccessController.doPrivileged((PrivilegedAction<Integer>) () -> Integer.getInteger("javafx.animation.pulse"));
+    private static final boolean multithreaded = initMultithreaded();
+    private static boolean initMultithreaded() {
```

Comment:
Have you considered a pattern such as this?

```java
    private static final boolean multithreaded = ((Supplier<Boolean>)() -> {
        // If it is not specified, or it is true, then it should
        // be true. Otherwise it should be false.
        String value = System.getProperty("quantum.multithreaded");
        if (value == null) return true;
        final boolean result = Boolean.parseBoolean(value);
        if (verbose) {
            System.out.println(result ? "Multi-Threading Enabled" : "Multi-Threading Disabled");
        }
        return result;
    }).get();
```

This would prevent a proliferation of many new init methods.

Reply:
That's an interesting suggestion: trading a new method for a lambda (the prior code used a lambda to pass to the doPrivileged, so it's also less of a delta). I'll take a look at it and see if it looks better to me.

Reply:
I decided to make this change. Thanks for the suggestion.

Reply:
This looks useful.
I wonder if it would make sense to create a utility method in some common Util class?

```
public static boolean initBoolean(BooleanSupplier c) { return c.getAsBoolean(); }
```

Reply:
We could, but I don't see enough value in doing it. The only thing you would save is the explicit cast to `(Supplier<Boolean>)` and the `.get()`.

Rather than that, it might be useful  (as a follow-up, not as part of the various doPriv removal PRs), to have a centralized utility method that also did the reading and parsing of the property. I'm fairly certain we already have some localized methods of this sort somewhere.

In any case, I don't want to make a refactoring change that relies on a new utility method as part of this effort

Reply:
ok